src/Process/Listener/MutationTestingResultsLoggerSubscriber.php

```diff
@@ -89,15 +89,8 @@ public function onMutationTestingFinished(MutationTestingFinished $event)
 
     private function filterLogTypes(array $logTypes): array
     {
-        $allowedFileTypes = [
-            ResultsLoggerTypes::TEXT_FILE,
-            ResultsLoggerTypes::DEBUG_FILE,
-            ResultsLoggerTypes::SUMMARY_FILE,
-            ResultsLoggerTypes::BADGE,
-        ];
-
-        foreach ($logTypes as $key => $value) {
-            if (!in_array($key, $allowedFileTypes, true)) {
+        foreach (ResultsLoggerTypes::ALL as $key => $value) {
```

Comment:
em, seems like `$logTypes` should be here. Isn't it?

Reply:
indeed, thank you